spec/system/dashboards/user_visits_dashboard_spec.rb

```diff
@@ -1,57 +1,137 @@
 require "rails_helper"
 
 RSpec.describe "Dashboard", type: :system, js: true do
-  let(:user) { create(:user) }
-  let(:listing) { create(:listing) }
+  let(:tag) { create(:tag) }
+  let(:organization) { create(:organization) }
+  let(:podcast) { create(:podcast) }
+  let(:listing) { create(:listing, user: collection.user) }
   let(:collection) { create(:collection, :with_articles) }
-  let(:collection_user) { collection.user }
+  let(:user1) { collection.user }
+  let(:user2) { create(:user) }
 
   context "when looking at analytics counters" do
     before do
-      sign_in user
+      sign_in user1
     end
 
-    it "shows the count of unspent credits" do
-      Credit.add_to(user, 2)
+    it "shows the count of unspent credits and listings created" do
+      Credit.add_to(user1, 2)
 
       Credits::Buy.call(
-        purchaser: user,
+        purchaser: user1,
         purchase: listing,
         cost: 1,
       )
       Credit.counter_culture_fix_counts
-      user.reload
+      user1.reload
 
       visit dashboard_path
 
       within "main#main-content > header" do
         expect(page).to have_text(/1\nCredits available/)
+        expect(page).to have_text(/1\nListings created/)
       end
     end
   end
 
   context "when looking at actions panel" do
```

Comment:
I'd appreciate some expert eyes on my spec approach here, as they take quite a while to run. 😅 

I am essentially testing all the counts in `Dashboard` for each of the 6 Dashboard paths (Big-O^2, anybody? 🙈 ).

I did consider testing just the `/dashboard` path, but then I remembered that [the PR](https://github.com/forem/forem/pull/17513) which motivated this current one had the correct Series count on `/dashboard` but NOT in any of the other Dashboard paths. 🤷🏾‍♀️